src/views/ward-leader.vue

```diff
@@ -293,6 +293,69 @@ export default {
     ]),
     sampleBallotFormPrefilled () {
       return `${SAMPLE_BALLOT_FORM}?ward=${this.ward}&party=${this.party}`
+    },
+    allCommitteePersons: function () {
+      let commPersons = this.committeePersons
+      // Get all divisions from ward boundaries
+      let allDivisions = this.wardBoundaries.features.map(x => x.properties.division)
+      // Default committe person data for Vacant divisions
+      let personDefault = function (ward, party) {
+        return {
+          'ward': ward,
+          'fullName': 'VACANT',
+          'division': '',
+          'party': party,
+          'address': '',
+          'id': '',
+          'zip': ''
+        }
+      }
+      let committeePersonsList = []
+      // Add placeholder objects for vacant divisions
+      let wardName = this.ward
+      if (hasSubWard(this.ward)) {
+        wardName = splitSubWard(this.ward).ward
+      }
+      let wardString = wardName.toString().padStart(2, '0')
+      let partyString = this.party.slice(0, 3).toUpperCase()
+      for (let a in allDivisions) {
+        let division = allDivisions[a]
+        let divisionId = `${wardString}-${division.toString().padStart(2, '0')}-${partyString}`
+        let subDivisions = ['A', 'B']
+
+        for (let b in subDivisions) {
+          let subDivision = subDivisions[b]
+          let subDivisionId = `${divisionId}-${subDivision}`
+          // Check for sub division id in ward leader data and add placeholder if missing
+          let personData = commPersons.find(c => c.id === subDivisionId)
+          if (personData === undefined) {
+            personData = personDefault(this.ward, this.party)
+            personData.id = subDivisionId
+            personData.division = division
+            personData.address = `Division ${subDivision}`
+          }
+          committeePersonsList.push(personData)
+        }
+      }
+      // Sort output by id
+      committeePersonsList.sort((a, b) => {
```

Comment:
nit, you could put the return on this line.

The sort function can also be a more compact form like:
`(a, b) => {
     if (a.id == b.id) {
       return 0;
    }
   return (a.id < b.id) ? -1 : 1;
}`